plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -97,7 +99,15 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
             }
         }
         finally {
-            thriftMetastore.releaseTableLock(lockId);
+            try {
+                thriftMetastore.releaseTableLock(lockId);
+            }
+            catch (Exception e) {
+                log.warn("Release lock step has failed for lockId: \"%s\". Not throwing the exception while releasing the lock, after commit has already succeeded. " +
```

Comment:
The recipient of the log file isn't the developer, so you don't need to explain what would happen if he exception was rethrown. This belongs to a code comment

Reply:
add comment, and reduce the exception message.